### DISCUSSION TITLE
How to load and use fragments from resources?

### DISCUSSION THREAD

**Discussion by tippfelher** [Netflix/dgs-framework#2052] [Q&A]
The resources folder in my java project contains schema graphqls files which is working.

However, I also have a fragments folder which contains... fragments. For example: test.graphql

```graphql
fragment MyEntityFields on MyEntity {
    id
    title
}
```

How can I use it in my `@Language("graphql")` annotated strings in my Java project?

When I try to use them using `...MyEntityFields`  the Editor picks them up but at runtime I am getting an error that the Fragment could not be found.

I also haven't found any repo or documentation that showcases how to use fragements. (can't embed them directly because `MyEntityFields` is used in like 50 places or so.. 😢

**Comment by hhravn** [comment]
@tippfelher did you make any progress on this? We're in the same situation here.
